react/src/components/molecules/Breadcrumb/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './style.css';
+
+const Breadcrumb = (props) => (
+  <ul className="ma__breadcrumb" role="navigation">
```

Comment:
<nav aria-label="Breadcrumb" class="breadcrumb">
  <ol>
  </ol>
</nav>

add aria-current="page" to last item

Reply:
<nav aria-label="Breadcrumb" class="breadcrumb">
  <ol>
  </ol>
</nav>